src/post/post.entity.unit.test.ts

```diff
@@ -212,6 +212,78 @@ describe('Post', () => {
             expect(note.content).toBe('<p>My first note</p>');
         });
 
+        it('creates a reply with mentions', () => {
+            const account = internalAccount();
+            const inReplyTo = Post.createNote(account, 'Parent');
+            // TODO: Clean up the any type
+            // biome-ignore lint/suspicious/noExplicitAny: Legacy code needs proper typing
+            (inReplyTo as any).id = 'fake-id';
```

Comment:
To avoid the `any`, we could do something like `(inReplyTo as unknown as { id: string }).id = 'fake-id';` - which is still a workaround, but at least avoid us adding more any to the codebase?

Reply:
Done in https://github.com/TryGhost/ActivityPub/pull/666/commits/ac30087a15cead75ae3a68a74bc6ae8eb5857d03